tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -76,49 +98,113 @@ object VarDictJava extends Configuration {
-    minimumQuality.foreach(buffer.append("-q", _)) // The minimum base quality for a "good call"
+    buffer.append("-z", "1") // Set to 1 since we are using a BED as input.
+    buffer.append("-c", "1") // The column for the chromosome.
+    buffer.append("-S", "2") // The column for the region start.
+    buffer.append("-E", "3") // The column for the region end.
+    buffer.append("-g", "4") // The column for the gene name.
+    if (!includeNonPf) buffer.append("-F", "0x700") // Ignore non-PF reads.
+    buffer.append("-f", minimumAf) // The minimum allele frequency threshold.
+    minimumAltReads.foreach(buffer.append("-r", _)) // Minimum # of reads supporting an alternate allele.
+    minimumQuality.foreach(buffer.append("-q", _)) // The minimum base quality for a "good call".
     maximumMismatches.foreach(buffer.append("-m", _)) // Maximum number of mismatches for reads to be included, otherwise they will be filtered.
-    if (countNsInTotalDepth) buffer.append("-K") // count Ns in the total depth ("DP" field)
+    if (countNsInTotalDepth) buffer.append("-K") // Count Ns in the total depth ("DP" field).
     buffer.append("-th", resources.cores.toInt) // The number of threads.
     buffer.append(bed)
 
     buffer
   }
 }
 
-/** Converts VarDictJava tabular output to valid VCF output. */
+/** Converts tumour/normal VarDictJava tabular output to valid VCF output. */
+private class Var2VcfPaired(tumorName: String,
+                            normalName: String,
+                            prefixContigsWithChr: Boolean = true,
+                            includeNonPfVariants: Boolean = false,
+                            outputOnlyCandidateSomatic: Boolean = false,
+                            allVariants: Boolean = true,
+                            maximumDistanceToRemoveHighQualitySnvPairs: Option[Int] = None,
+                            maximumNonMonomerMsi: Option[Int] = None,
+                            maximumMeanMismatches: Option[Double] = None,
+                            maximumPValue: Option[Double] = None,
+                            minimumMeanVariantPositionInReads: Option[Double] = None,
+                            minimumMeanBaseQuality: Option[Double] = None,
+                            minimumMappingQuality: Option[Double] = None,
+                            minimumDepth: Option[Int] = None,
+                            minimumHighQualityAltDepth: Option[Int] = None,
+                            minimumAf: Double = VarDictJava.MinimumAf,
+                            minimumSignalToNoiseRatio: Option[Double] = None,
+                            minimumHomozygousAlleleFreq: Option[Double] = None
+                           ) extends ProcessTask with PipeWithNoResources[Text, Vcf] {
+  name = "Var2VcfPaired"
+
+  /** Format the two sample names into one argument by joining on the default separator. */
+  private def sampleNames: String = {
+    Seq(tumorName, normalName).mkString(VarDictJava.TumorNormalSeparator)
```

Comment:
Do you want to maybe add an assert here that neither sample name already contains `TumorNormalSeparator`?  I could see that causing havoc.